package.json

```diff
@@ -4,13 +4,15 @@
   "license": "MIT",
   "scripts": {
     "start": "node scripts/start.mjs",
+    "start-https": "node scripts/start.mjs --https=true",
```

Comment:
we'll want to add this to the README, but we probably need to do a larger revamp there!

Reply:
thanks for flagging, I was just looking at changing the docs in `blocks` but I forgot there's a mention in the READMEs too. I think maybe we should change the READMEs to point to the docs, wdyt?

Reply:
I think I agree, although this might be a template-specific detail? So potentially both, as painful as that is?

Reply:
I feel like there's nothing in the template README that isn't covered in the docs on building a block (at https://blocks.githubnext.com/githubnext/blocks/blob/main/docs/Developing%20blocks/2%20Building%20your%20first%20block.md) so I'm inclined to make the READMEs just a pointer, unless you see a reason to keep the existing quickstart (at https://github.com/githubnext/blocks-template-svelte/#readme).

Reply:
I'm sure I'm missing something! I was thinking about things like this specific `start-https` npm script - something that might not be present in eg. a community-created Blocks template. But if it's in all of our "official" ones, maybe not an issue at all

Reply:
oh hm I wasn't thinking of community-created templates, I think you're saying the official docs wouldn't cover those.

OK let's punt for now, I will update the READMEs and the main docs.